cmd/rookflex/cmd/mount.go

```diff
@@ -153,7 +153,9 @@ func mountDevice(client *rpc.Client, mounter *k8smount.SafeFormatAndMount, devic
 		)
 		if err != nil {
 			log(client, fmt.Sprintf("mount volume %s/%s failed: %v", opts.BlockPool, opts.Image, err), true)
-			os.Remove(globalVolumeMountPath)
+			if err := os.Remove(globalVolumeMountPath); err != nil {
+				return errors.Wrapf(err, "failed to remove dir %s", globalVolumeMountPath)
```

Comment:
The changes in this file all return the original error on the next line, which is preferred. Let's just log the errors in this file instead of returning them.

Reply:
Also in this file, we need to log with the `log()` method. Inside the flex driver we actually can't use the `logger.Infof()` and similar logging methods.

Reply:
Done.

Reply:
Did you push the changes? I still see errors returned instead of calling `log()`